db/migrate/20230116201724_refactor_group_membership.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class RefactorGroupMembership < ActiveRecord::Migration[6.1]
+  def up
+    drop_table :group_memberships
```

Comment:
You are right the existing data should be handled.
I renamed the table and model (instead of deleting it) and added a migration for the data
Table and model (and groupify) should be removed in their own PR afterwards

Reply:
Thanks for taking care. I just have some concerns because we rely on the Gem for the migration, which we are about to remove, and because the model class is not part of the migration (which makes it difficult to use the migration later, if we cleaned up). Therefore, I have two questions:

1. Can we drop relying on Groupify for the migration? It looks like you mainly use the column values directly, and rather not rely on Groupify any longer.
2. Can you then move the temporary Active Record to the migration? Here is an excerpt from the Rails Style Guide, providing an example: https://rails.rubystyle.guide/#define-model-class-migrations

Reply:
Very nice solution!